### PR TITLE
Templates fixes

### PR DESCRIPTION
**Pull request by mrbuds**
> fix totem glow
> typo in a description
> new "overlayGlow" subType for spells with procs
> added and fixed a few spells

### CODE REVIEW

File: WeakAurasTemplates/TriggerTemplatesData.lua [hunk 1]
```diff
@@ -158,22 +158,28 @@ templates.class.WARRIOR = {
       args = {
         { spell = 100, type = "ability", requiresTarget = true, talent = {5,6}}, -- Charge
         { spell = 100, type = "ability", charges = true, requiresTarget = true, talent = 4, titleSuffix=" (2 Charges)"}, -- Charge
+        { spell = 121, type = "ability", requiresTarget = true}, -- Slam
```
Comment (InfusOnWoW) [inline WeakAurasTemplates/TriggerTemplatesData.lua]: Unknown spell id: 121 and 129.

Comment (mrbuds) [reply WeakAurasTemplates/TriggerTemplatesData.lua]: oops! I typed macro id there, it's fixed, and i also renamed Cooldowns to Abilities as you said on discord

**Comment by InfusOnWoW** [comment]
I'm not sure whether Abilities is the best, but imho it's better than Cooldowns

**Comment by Stanzilla** [comment]
Question, I noticed some templates have an unchecked "inverse" condition set, what is that used for exactly?

**Comment by mrbuds** [comment]
"inverse" is checked by default because when a cooldown start we can't use it and we show when it will be ready 
cd
But when we show auras duration, we want the icon spiner to show we have it, and when it will be off : 
aura cd

**Pull request merged**